requirements.txt

```diff
@@ -44,3 +44,7 @@ webcolors==1.4
 Django==1.8
 djangorestframework==3.1.1
 django-rest-swagger==0.3.2
+
+# JSON API Formatting
+git+https://github.com/pattisdr/django-rest-framework-json-api.git@develop
```

Comment:
Best to pin to a commit # rather than @ develop because it will keep pip from trying to re-download it each time. The downside is that you will have to update the commit number whenever you make a change to the branch in order to get the change to go to the OSF code, but I'm generally of the mindset that those changes should be brought in explicitly. That way silent changes to the OSF codebase can't happen.